docs/contributing/translation/style-guide.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
Has this filed moved? 🤔 
I thought this file was already translated

Reply:
Correct -- it used to be here but got moved in this PR: https://github.com/gatsbyjs/gatsby/pull/21254

There's a couple of changes to content so it would be worth it to re-translate it and the related pages.